src/client/eppo-client.ts

```diff
@@ -133,7 +172,17 @@ export interface IEppoClient {
     configurationRequestParameters: FlagConfigurationRequestParameters,
   ): void;
 
-  setConfigurationStore(configurationStore: IConfigurationStore<Flag | ObfuscatedFlag>): void;
+  setFlagConfigurationStore(configurationStore: IConfigurationStore<Flag | ObfuscatedFlag>): void;
+
+  setBanditVariationConfigurationStore(
+    banditVariationConfigurationStore: IConfigurationStore<BanditVariation[]>,
+  ): void;
+
+  setBanditModelConfigurationStore(
+    banditModelConfigurationStore: IConfigurationStore<BanditParameters>,
+  ): void;
+
+  setIsObfuscated(isObfuscated: boolean): void;
```

Comment:
` setIsObfuscated()` wasn't added to the `IEppoClient` interface so figured I'd hoist it up along with the above newly-added bandit store setters 🧹